docs/source/contributing.rst

```diff
@@ -35,7 +35,7 @@ This project uses git for version control and github for issue tracking. If you
 Creating a Fork
 ---------------
 
-Once you have your git credentials setup, the next step is to create a fork of the project to work off of.
+Once you have your git credentials setup using this `tutorial <https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-ssh-urls>`_, the next step is to create a fork of the project to work off of.
```

Comment:
This doesn't look like the best link - you don't need SSH at all for this. I think the sentence essentially means "have an account on github", i.e., to have signed up.

Reply:
Since the documentation didn't specify the https url for cloning the repo, I thought of adding this reference.

Reply:
That should then go with a separate instruction to clone the repo - which is a couple of lines below.